### ISSUE TITLE
NVDA fails to read with mouse tracking post comments in Facebook with Microsoft Edge

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#9111]
### Steps to reproduce:
1. Enable mouse tracking in NVDA. My mouse unit resolution is set to "Paragraph", but I don't think it matters in this case.
2. Open Microsoft Edge.
3. Go to www.facebook.com.
4. Log in with your Facebook account.
5. Try to read post comments in your news feed with NVDA's mouse tracking/echo feature.

### Actual behavior:
NVDA says only "comment" for each post comment.

### Expected behavior:
NVDA should read the actual content of the post comments.

### System configuration
- Operating system: Windows 10 Pro version 1809 (build 17763.195), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-16477,f6cd02d2.
- Microsoft Edge version: Microsoft Edge 44.17763.1.0, Microsoft EdgeHTML 18.17763.
- Processor: Intel Core i5-4670 at 3.40GHz.
- RAM Memory: 8.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC887 at Intel Lynx Point PCH - High Definition Audio Controller.

#### Does the issue still occur after restarting your PC?
Yes, it does.

#### Have you tried any other versions of NVDA?
Yes, v2018.4 and the result is the same.

**Comment by Adriani90** [comment]
@k-kolev1985 can you also reproduce this in Edge Chromium 81? If not, I vote for closing this issue since Edge Chromium will be distributed to most users in Windows update 20 H1.

**Comment by k-kolev1985** [comment]
@Adriani90 No, I cannot reproduce this in Chromium Edge 81.

**Comment by Adriani90** [comment]
Then I would close this for now since Edge Cromium will be distributed with the Windows 10 Update 20 H1 to most users.

**Comment by Adriani90** [comment]
Adding appropriate label since this is probably still reproducible in Edge Spartan and UIA related environments.
